src/ztex/device.c

```diff
@@ -92,7 +92,7 @@ static int device_init_fpgas(struct device *device,
 {
 	// Read config for given bitstream, device
 	char *CFG_SECTION = "ZTEX:";
-	char conf_name_board_freq[256], conf_name_freq[256];
+	char conf_name_board_freq[11 + ZTEX_SNSTRING_LEN], conf_name_freq[256];

```

Comment:
I can make it 128 + ZTEX_SNSTRING_LEN. That leaves enough bytes to change the string later. To avoid the truncation warning conf_name_boad_freq must be a few bytes shorter than conf_name_freq. Or it might be even better to just set it to 128, if alignment is an issue.